asv_bench/benchmarks/plotting.py

```diff
@@ -161,4 +161,45 @@ def time_get_plot_backend_fallback(self):
         _get_plot_backend("pandas_dummy_backend")
 
 
-from .pandas_vb_common import setup  # noqa: F401 isort:skip
+class DataFramePlottingLarge:
```

Comment:
I think what differentiates this from `FramePlotting` is that these are wide. Can you rename this class to `FramePlottingWide`.